frontends/p4/simplify.cpp

```diff
@@ -33,7 +37,11 @@ const IR::Node* DoSimplifyControlFlow::postorder(IR::BlockStatement* statement)
         return statement;
     }
     bool withinBlock = statancestor != nullptr && statancestor->is<IR::BlockStatement>();
+
+    // TODO: withinAction will always be false: If parent is a P4Action, the if
```

Comment:
This should probably be replaced with `findContext<IR::P4Action>() != null`.
I will file an issue about this.